chamge/amqp-edge-backend.c

```diff
@@ -7,19 +7,73 @@
 #include "config.h"
 
 #include "amqp-edge-backend.h"
+#include "amqp/amqp-rpc.h"
+
+#include <gio/gio.h>
+
+#define AMQP_EDGE_BACKEND_SCHEMA_ID "org.hwangsaeul.Chamge1.Arbiter.AMQP"
```

Comment:
The id may be `org.hwangsaeul.Chamge1.Edge.AMQP`?

What I am thinking here is that *edge* shouldn’t know *arbiter* vice versa. 
they are solely independent.

Reply:
I know it. I was just lazy. ^^; I will fix it, too.